### DISCUSSION TITLE
done running the command provided, but getting this error.

### DISCUSSION THREAD

**Discussion by DARK7BEA5T** [tribalfs/PixelsPub#86] [General]
Exception occurred while executing 'grant':
java.lang.SecurityException: grantRuntimePermission: Neither user 2000 nor current process has android.permission.GRANT_RUNTIME_PERMISSIONS.

**Comment by tribalfs** [comment]
Check notes in [steps 2 and 7](https://github.com/tribalfs/Pixels#2-enable-usb-debugging)

**Reply by DARK7BEA5T** [reply]
i think your code has ome error, cuz i did all step.

**Reply by DARK7BEA5T** [reply]
i downloaded purr, used its code, it worked, but not change resolution, then i used its code, just change app package to this software's, it work.

**Reply by tribalfs** [reply]
Probably you just mistyped the command.

**Reply by DARK7BEA5T** [reply]
i copy pasted. it cant not be mistyped

**Reply by DARK7BEA5T** [reply]
code is same, yeah maybe it was some prolem with adb, cus i closed cd before doing it again

**Comment by tommiehansen** [comment]
Solution is to also enable USB debugging for security stuff.

The confusion is due to the fact that this isbt mentioned in the setup for users not using a PC.

As @tribalfs mentions this is mentioned in the documentation but only if using a PC, not when using local ADB ie LABD. A simple note in the docs should clear it up.

**Comment by ffhirochima18-spec** [comment]
> Exception occurred while executing 'grant': java.lang.SecurityException: grantRuntimePermission: Neither user 2000 nor current process has android.permission.GRANT_RUNTIME_PERMISSIONS.

https://github.com/tribalfs/PixelsPub/commit/9dbed099204f4e43e1220c4307abe13ced8b4f94#diff-728436c5d6842d3b426f82c1b0468dce9d9c7ac50a47cab5ee461acc3004dc83L2

**Comment by ffhirochima18-spec** [comment]
Free faire

**Comment by fernandocardosoblek-crypto** [comment]
9dbed09#diff-728436c5d6842d3b426f82c1b0468dce9d9c7ac50a47cab5ee461acc3004dc83L2

**Comment by fernandocardosoblek-crypto** [comment]
adb shell pm grant com.tribalfs.pixels android.permission.WRITE_SECURE_SETTINGS

**Comment by fernandocardosoblek-crypto** [comment]
Kn7vruiob8bd5csww89n8bkt8b7fs90b8b7v6cs3vopn7e288g6ct5xsu8v7vusu8vettyic6f6c6f6ffg6dw68kllgsaghbvggc
